lib/request.js

```diff
@@ -85,12 +98,10 @@ module.exports = function (protocol, host, version, token) {
                 Authorization: 'bearer ' + token
             };
         }
-        return makeRequest(object, function (err, result) {
-            if (err) {
-                console.log('error happen: ' + err);
-            }
-
-            callback(err, result);
-        });
+        if (namespace){
```

Comment:
Add support for namespacing at the individual collection request level - openshfit namespaces all it's resources, so it was useful to be able to do this at the parent client level.